cmd/snap/cmd_snap_op.go

```diff
@@ -932,7 +936,7 @@ func init() {
 			// TRANSLATORS: This should not start with a lowercase letter.
 			"revision": i18n.G("Refresh to the given revision, to which you must have developer access"),
 			// TRANSLATORS: This should not start with a lowercase letter.
-			"list": i18n.G("Show available snaps for refresh but do not perform a refresh"),
+			"list": i18n.G("Show the snaps that would be updated on refresh, but do not perform the refresh"),
```

Comment:
Might it be worth adding that the output lists the store version and revision, rather than  the version installed (which is unclear, I think)?
eg. `Shows the new versions of snaps that would be updated with the next refresh.`